includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -958,38 +980,6 @@ static function() {
 		);
 	}
 
-	/**
-	 * Override the featured image header styling in style.css.
-	 * Used only for Twenty Seventeen.
-	 *
-	 * @since 1.0
-	 * @link https://github.com/WordPress/wordpress-develop/blob/1af1f65a21a1a697fb5f33027497f9e5ae638453/src/wp-content/themes/twentyseventeen/style.css#L2100
-	 */
-	public static function add_twentyseventeen_image_styles() {
```

Comment:
This seemed to actually _worsen_ the featured image header styling:

> ![image](https://user-images.githubusercontent.com/134745/128580065-ac9dab13-508d-43a3-bdc6-aaf4164de9a9.png)

Removing the style fixes the vertical positioning and removes that grayness:

> ![image](https://user-images.githubusercontent.com/134745/128580084-5cadf110-17c3-4158-b347-6e2ce983267f.png)

Reply:
Assuming the left is non-AMP and the right AMP, I'm confused as to why in both instances the featured image look the same. Wouldn't the styles only affect the AMP version, and thus a difference on the AMP page?

Reply:
Sorry, yes, the left version is the non-AMP and the right is the AMP.

In the first screenshot (before) you can see this slight gray bar under the image. This goes away and the image is identical to the non-AMP version when these styles are removed. So it seems the original reason for this code being add in the first place was eliminated at some point, either through a change in AMP core or to the Twenty Seventeen theme. 

In fact, what confuses me is that this is overriding the display to be `inline-block` whereas the GitHub link indicates that the theme has `block`. The issue may have been fixed with the introduction of `disable-inline-width` in https://github.com/ampproject/amp-wp/pull/2036.